packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js

```diff
@@ -126,10 +127,18 @@ const ProductFullDetail = props => {
     }
 
     const cartActionContent = isSupportedProductType ? (
-        <Button disabled={isAddToCartDisabled} priority="high" type="submit">
+        <Button
+            disabled={isAddToCartDisabled || isOutOfStock}
```

Comment:
```suggestion
            disabled={isAddToCartDisabled}
```
I dont think we need the OR condition because [you have updated](https://github.com/magento/pwa-studio/pull/3356/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-37a9126fa04a6065c2291b60d9ec7d01e0cb85bf0d8873d1c894058441c24023R473) the talon's `isAddToCartDisabled` definition to check for out of stock status.

Reply:
Removed condition.